.npmrc

```diff
@@ -0,0 +1 @@
+shamefully-hoist=true
```

Comment:
create a flat node modules - make build-react-ui-components-standalone will fail because react dnd has unrequired dependencies and more
better use https://github.com/pnpm/pnpm/issues/1262#issuecomment-683351769